### DISCUSSION TITLE
Do we have to account for a time-shift after fed dosing?

### DISCUSSION THREAD

**Discussion by tobiasK2001** [Open-Systems-Pharmacology/Forum#872] [Questions & Problems] [answered]
Dear Community,

please share your opinion for the following problem:
when we simulating fed clinical studies, we can add a food event prior to or simultaneous with drug dosing. In study protocols  you typically find statements like:
"a breakfast was served 30 min before drug administration". In PK-sim you cannot add a food event with negative times. So I wonder if we should design the simulation either by include a food event at t=0 and the drug administration at t=30 ( = S1). In that case to compare the simulation to measured data, the time axis of the data has to be transformed accordingly.
Or would it be enough to input both events at t=0 ( = S2) use the data as is and ignore a time difference between food intake and drug dosing.
To check if there could be some deviation expected, I simulated a typical BCS Class2 compound with the scenarios S1 and S2. To directly compare S1 with S2 I esported the simulation results, shifted the time 30 min ahead and reimported the result for a direct overlay.
The results are as expected slightly different.
![image](https://user-images.githubusercontent.com/25585248/109141753-616b8680-775e-11eb-9f09-ceb317048055.png)
 The differences are somewhat visible but small if looking at the deviation especially for t-max and Cmax:

  | S1 Tabled-fed_dosing at t=30 | S2 Tabled-fed_dosing and food   event at t=0 | S2 with time-shift | deviation S2 vs S1 %
-- | -- | -- | -- | --
AUC_inf [µmol*min/l] | 319.645 | 319.998 | 319.998 | 0.11%
AUC_tEnd [µmol*min/l] | 315.066 | 315.440 | 315.440 | 0.12%
C_max [µmol/l] | 0.218 | 0.213 | 0.213 | -2.41%
C_tEnd [µmol/l] | 0.002 | 0.002 | 0.002 | -0.52%
Total body clearance/F [ml/min/kg] | 2.012 | 2.009 |   |  
MRT [h] | 35.280 | 34.997 | 35.497 | 0.61%
t_max [h] | 3.000 | 2.750 | 3.250 | 8.33%
Half-Life [h] | 31.558 | 31.573 | 31.574 | 0.05%

 However, I am wondering if the difference justifies the changed simulation design and the extra effort to modify the measured data? Or can it just be ignored? As there is to much interindividual variability in measured fed data anyway?

Please shre your thoughts on this

Best, 
Tobias

**Comment by StephanSchaller** [comment, marked as answer]
@tobiasK2001 

My thoughts would be that the characterization (and exact knowledge) of food composition and the exact way it is then consumed is too uncertain that the exact time of the start of food consumption matters in your case example.

This might be different if you have explicit food effects (as we often have in e.g. diabetes where food uptake is then related to nutrient (e.g. glucose) appearance in the system).

So it is really a case-by-case decision depending on the relevance of (i.e. sensitivity of your scenario towards) food effects.

**Comment by PavelBal** [comment]
This feature request would help to deal with offests in observed data: https://github.com/Open-Systems-Pharmacology/MoBi/issues/161

Maybe you can upvote it if you also think it is useful ;)

**Comment by tobiasK2001** [comment]
Dear Stephan,

yes I think you have a point there. 
Thanks Pavel for the feature request.

Does anyone else have additional thoughts?

Best, Tobias
